examples/transaction_lifecycle_example/main.py

```
@@ -0,0 +1,179 @@
+            
+            # Extract the actual records from the CacheAwareResponse
+            records = response.data
+            
+            print(f"✅ Retrieved {len(records)} records from stream:")
+            for i, record in enumerate(records, 1):
+                # EventTime is returned as string, convert to int first
+                timestamp = int(record["EventTime"])
+                date = datetime.fromtimestamp(timestamp, tz=timezone.utc)
+                print(f"   Record {i}: {record['Value']} (Time: {date.strftime('%Y-%m-%d %H:%M:%S')})")
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)